src/App.css

```diff
@@ -0,0 +1,9 @@
+* {
+    box-sizing: border-box;
+}
+
+.emailCardContainer {
+    display: flex;
+    flex-direction: row;
+    height: 90vh;
+}
```

Comment:
this styling should probably be in the css file for that component?

Reply:
also height 90vh is suspicious 🤔 should probably be 100% of something instead, 90vh seems very particular

Reply:
oh i see, this is not a component, still `emailCardContainer made me think it was something else. maybe just make it a `main` tag?